admin_guide/managing_networking.adoc

```diff
@@ -1201,3 +1201,57 @@ objects:
             name: default
 ...
 ----
+
+[[admin-guide-enabling-hsts]]
+== Enabling HTTP Strict Transport Security
+
+HTTP Strict Transport Security (HSTS) policy is a security enhancement, which
+requests that traffic to the host be HTTPS. This is useful for ensuring secure
+interactions with websites, or to offer secure application for the user's
+benefit.
+
+The default behavior is to drop any HTTP requests that require HTTPS. Enabling
+HSTS lets you set up a redirect so that the first time a client requests the
+target it is redirected to HTTPS. Any subsequent attempts will get a HTTPS
+header in the response, meaning future requests will be HTTPS.
+
+HSTS adds a Strict Transport Security header to HTTPS responses from the site.
+The client (often a browser) reads the header and, from then on, can enforce the
+policy. All requests to the HTTP URL are changed to HTTPS before the request is
```

Comment:
Might want to make it clear that the client changes from http to https.

Reply:
Also, the client is not required to support this and max-age=0 turns it off.